Gemfile.lock

```diff
@@ -371,6 +373,7 @@ DEPENDENCIES
   brakeman
   debug
   discordrb (~> 3.5.0)
+  dockerfile-rails (>= 1.6)
```

Comment:
let's update bundler version in the `bundled with` block to 2.5.18, because it is used in docker image

Reply:
https://github.com/aifoundry-org/blame-ai-discord-bot/pull/23/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR391